docs/UserGuide.md

```diff
@@ -85,10 +85,16 @@ Adds a person to the address book.
 
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
-<div markdown="span" class="alert alert-warning">**Caution:**
+<div markdown="span" class="alert alert-warning">
+**Names:**
 Names must be alphanumeric and all contacts must have unique names.
+Names with symbols like "s/o" and "d/o" will not be allowed. Please write names without these symbols.
+Names with additional white spaces in between words will be taken as it is.
 
 The application will throw an error if you attempt to create two users with the same names
+
+**Phone Numbers:**
+Only Singapore phone numbers are allowed by the application. Meaning, that only phone numbers starting with the number 6, 8 or 9 and is exactly 8 digits in total is allowed.
```

Comment:
Small grammatical mistake "with the number 6, 8 or 9 and **are** exactly 8 digits in total **are** allowed". Though we may drop this feature so TBD

Reply:
Update: As suggested by Kenneth, can leave this in pending the grammatical mistakes being fixed and specify that international number support is coming in a future feature. "While we are aware of the possibility of international numbers, for this iteration, we decided to focus on the local population"